src/util/getTotal.ts

```diff
@@ -1,17 +1,19 @@
 import Big from 'big.js'
 
-export function getTotal<T, Name extends keyof T>(collection: T[], propName: Name): string {
-  let reduced: Big = collection.reduce((total: Big, item: T) => {
+export function getTotalBig<T, Name extends keyof T>(collection: T[], propName: Name): Big {
+  return collection.reduce((total: Big, item: T) => {
     let propValue = item[propName]
 
-    if (typeof propValue !== 'number') {
+    if (typeof propValue !== 'number' && typeof propValue !== 'string') {
       return total
     }
 
     return total.plus(propValue)
```

Comment:
Nit: I think it reads slightly better with:
```suggestion
    if (typeof propValue === 'number' || typeof propValue === 'string') {
      return total.plus(propValue)
    }

    return total
```

Reply:
Yeah we can do it this way!

Reply:
Actually we can't since Typescript doesn't like it. We'd have to check `propValue` much more.